coil-svg/src/jvmTest/kotlin/coil3/decode/SvgDecoderTest.jvm.kt

```diff
@@ -0,0 +1,149 @@
+package coil3.decode
+
+import coil3.Image
+import coil3.ImageLoader
+import coil3.fetch.SourceFetchResult
+import coil3.request.Options
+import coil3.size.Dimension
+import coil3.size.Scale
+import coil3.size.Size
+import coil3.test.utils.assertIsSimilarTo
+import coil3.test.utils.context
+import coil3.test.utils.decodeBitmapAsset
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertIs
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+import kotlinx.coroutines.test.runTest
+import okio.BufferedSource
+import okio.FileSystem
+import okio.Path.Companion.toPath
+import okio.buffer
+
+class SvgDecoderTestJvm {
+
+    private val decoderFactory = SvgDecoder.Factory()
+
+    @Test
+    fun handlesSvgMimeType() {
```

Comment:
Is it possible to implement these tests in a common way? If you can't put a common test class in `jvmCommonTest`, you could create `AbstractSvgDecoderTest` in `jvmCommonTest` which defines all the tests then have `SvgDecoderTestAndroid` and `SvgDecoderTestJvm` extend that class.

Reply:
I was trying to do this, but I can't access `jvmCommonTest` or even `commonTest` from `androidInstrumentedTest`.